README.md

```diff
@@ -10,14 +10,19 @@ Service APIs implemented:
 - Amazon SimpleDB
 - Amazon Mechanical Turk
 - Amazon CloudWatch (MON)
-- Amazon DynamoDB (ddb2) 
+- Amazon DynamoDB (ddb2)
 
-All API functions have been implemented.  Not all functions have been thoroughly tested, so exercise care when integrating this library into production code.  Please send bug reports and patches.
+All API functions have been implemented.  Not all functions have been
```

Comment:
Yep, but two or more whitespaces at the end of a line will be converted to a new line.